github-repos/src/components/ReposList.js

```diff
@@ -0,0 +1,29 @@
+import React, { Fragment, Component } from "react";
+import { connect } from "react-redux";
+
+class ReposList extends Component {
+  render() {
+    const { repos } = this.props;
+    return (
+      <Fragment>
+        <ul>
+          {repos.map(repo => (
+            <li key={repo.id}>{repo.name}</li>
+          ))}
+        </ul>
+      </Fragment>
+    );
+  }
+}
+
+const mapStateToProps = state => {
+  //   console.log("state :", state);
+  return {
+    repos: state.repos
+  };
+};
+
+export default connect(
+  mapStateToProps,
+  null
```

Comment:
Hi @prateekinstillion , is it really a good idea to export `null`?

Reply:
Hi Abhinav, I am not exporting null, the concept here is that since there are no actions to dispatch in this component, therefore second parameter to the connect function is set to null.